diff --git a/bin/genieacs-cwmp b/bin/genieacs-cwmp
index 7dd92d9..6b92420 100755
--- a/bin/genieacs-cwmp
+++ b/bin/genieacs-cwmp
@@ -2,6 +2,8 @@
 var path = require('path');
 var fs = require('fs');
 
+process.title = path.basename(__filename);
+
 var dir = path.resolve(path.dirname(fs.realpathSync(__filename)), '..');
 process.chdir(dir);
 
diff --git a/bin/genieacs-fs b/bin/genieacs-fs
index 5d0253b..8278d98 100755
--- a/bin/genieacs-fs
+++ b/bin/genieacs-fs
@@ -2,6 +2,8 @@
 var path = require('path');
 var fs = require('fs');
 
+process.title = path.basename(__filename);
+
 var dir = path.resolve(path.dirname(fs.realpathSync(__filename)), '..');
 process.chdir(dir);
 
diff --git a/bin/genieacs-nbi b/bin/genieacs-nbi
index 5069105..275ed6a 100755
--- a/bin/genieacs-nbi
+++ b/bin/genieacs-nbi
@@ -2,6 +2,8 @@
 var path = require('path');
 var fs = require('fs');
 
+process.title = path.basename(__filename);
+
 var dir = path.resolve(path.dirname(fs.realpathSync(__filename)), '..');
 process.chdir(dir);

